plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineProjectWizard.java

```diff
@@ -107,8 +115,12 @@ public boolean performFinish() {
     try {
       getContainer().run(fork, cancelable, runnable);
       
+      // prompt to switch to preferred perspective
+      BasicNewProjectResourceWizard.updatePerspective(configElement);
+
       // open most important file created by wizard in editor
       IFile file = runnable.getMostImportant();
+      BasicNewProjectResourceWizard.selectAndReveal(file, workbench.getActiveWorkbenchWindow());
```

Comment:
Could use `org.eclipse.ui.wizards.newresource.BasicNewResourceWizard.selectAndReveal(IResource)` instead.
Or at least change the class to `BasicNewResourceWizard`.